src/examples/device_in_label_group.js

```diff
@@ -0,0 +1,33 @@
+import Testdroid from 'testdroid-client';
+
+async () => {
+  let baseUrl = process.argv[2];
```

Comment:
nit: see above also might be worth seeing what I did in tc-vcs the pattern here looks very similar.